pkg/plugin/v2/scaffolds/edit.go

```diff
@@ -56,21 +61,26 @@ func (s *editScaffolder) Scaffold() error {
 			str,
 			"COPY api/ api/",
 			`COPY apis/ apis/`)
-		if err != nil {
+		if err != nil && s.multigroup != config.MultiGroup {
 			return err
 		}
 	} else {
 		str, err = ensureExistAndReplace(
 			str,
 			"COPY apis/ apis/",
 			`COPY api/ api/`)
-		if err != nil {
+		if err != nil && s.multigroup != config.MultiGroup {
```

Comment:
We can centralize the error check after the if/else

Reply:
done